packages/firebase_core/lib/src/firebase_app.dart

```diff
@@ -43,7 +43,7 @@ class FirebaseApp {
     return channel.invokeMethod('FirebaseApp#configure', <String, dynamic>{
       'name': name,
       'options': options.asMap,
-    }).then((Null _) => _namedApps[name]);
+    }).then((dynamic _) => _namedApps[name]);
```

Comment:
Do we need to specify a type here at all?

Reply:
unfortunately yes